qdrant-landing/content/documentation/cloud/authentication.md

```diff
@@ -5,31 +5,46 @@ weight: 30
 
 # Database Authentication in Qdrant Managed Cloud
 
-This page shows you how to use the Qdrant Cloud Console to create a Database API key for a cluster. You will learn how to connect to your cluster using the new API key.
+This page describes what Database API keys are and shows you how to use the Qdrant Cloud Console to create a Database API key for a cluster. You will learn how to connect to your cluster using the new API key.
+
+Database API keys can be configured with granular access control. Please refer to the [Table of access](/documentation/guides/security/#table-of-access) to understand what permissions you can configure.
```

Comment:
Should Table be capitalised if so - should Access be capitalised?

Reply:
I'm using it as it used on the referenced page, it is named `Table of access` there. Let me know if you want it to be changed.